src/libdrakvuf/drakvuf.c

```diff
@@ -159,6 +174,20 @@ bool drakvuf_init(drakvuf_t* drakvuf, const char* domain, const char* rekall_pro
     if ( !xen_init_interface(&(*drakvuf)->xen) )
         goto err;
 
+    PRINT_DEBUG("drakvuf_init: building event channel\n");
+    (*drakvuf)->evtchn = xc_evtchn_open(NULL, 0);
```

Comment:
This should be moved into the drakvuf->xen interface structure and created by `xen_helper` during `xen_init_interface`